src/components/UI/Home_Header.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../styleSheets/_variables.scss";
+@import "../../styleSheets/_mixins.scss";
+@import "./Home.scss";
+
+#background {
+  @include background-image("src/assets/images/castles/iceTower.jpg");
+  height: 100vh;
+
+  #home-header {
+    @include flex(flex-end, center, row);
+    background-color: rgba($color: #000000, $alpha: 0.5);
+    padding: $padding-4;
```

Comment:
glad to see you using the variables. i dont think i did that fully yet but awesome! i will get on that band wagon aswell :)